src/tools/execution-results.h

```diff
@@ -202,6 +223,9 @@ struct ExecutionResults {
       // call the method
       for (const auto& param : func->sig.params) {
         // zeros in arguments TODO: more?
+        if (!param.isDefaultable()) {
+          std::cout << "[trap fuzzer cannot provide a non-null export param'\n";
```

Comment:
What is `export param`? You mean a param to an export function?

Reply:
Yes, sorry, I did mean a param to an export. I'll clarify.